samples/boards/nrf52/mesh/onoff_level_app/src/common.c

```diff
@@ -0,0 +1,128 @@
+#include "common.h"
+
+struct nvs_fs fs = {
+	.sector_size = NVS_SECTOR_SIZE,
+	.sector_count = NVS_SECTOR_COUNT,
+	.offset = NVS_STORAGE_OFFSET,
+	.max_len = NVS_MAX_ELEM_SIZE,
+};
+
+struct device *led_device[4];
+struct device *button_device[4];
+
+static struct k_work button_work;
+
+static void button_pressed(struct device *dev,
+			   struct gpio_callback *cb, uint32_t pins)
+{
+	k_work_submit(&button_work);
+}
+
+void gpio_init(void)
+{
+	static struct gpio_callback button_cb[4];
+
+	/* --------------- LEDs configiuratin & setting --------------- */
```

Comment:
Can you get rid of these `/* ----- ----*/` lines? I don't think we use them elsewhere in Zephyr, so trying to keep the code consistent